### PR TITLE
updated commons codec, removed efs publishing code, updated readme

### PR DESCRIPTION
**Pull request by avalaw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ with an authorization response.
 
 ## Building the SDK ##
 ### Everlaw ###
-In the `servers` repository, the root-level `pom.xml` should have sharefile-api listed as a dependency with the current version. It should also be listed in the `pom.xml` for `web`, without version. On build, it will look for the artifact in your local maven repository, or in the archiva repository on EFS if it doesn't find it locally.
+In the `servers` repository, the root-level `pom.xml` should have sharefile-api listed as a dependency with the current version. It should also be listed in the `pom.xml` for `web`, without version. On build, it will look for the artifact in your local maven repository, or in github packages if it doesn't find it locally.
```
Comment (amyli123) [inline README.md:29]: Was the previous doc just out of date, or was this an actual change that was made?

Comment (avalaw) [reply README.md:29]: The previous readme was way out of date, I don't think the build system has used EFS for this package in years. The sharefile sdk package was uploaded to GH packages in 2021. We might not even be using EFS for anything in the build process at all anymore

**Comment by amyli123** [comment]
Is there a reason we're targeting `master`?

**Comment by avalaw** [comment]
@amyli123 I don't think there's any kind of git branching strategy in place for this repo, it hasn't been touched in years, so I just opened a PR to master. Open to suggestions though.

It's worth noting, also, that the `servers` build process doesn't pull in this repo directly, it uses artifacts stored in GH Packages, and for this package it's calling for a specific version number. So merging to master, and even pushing up the artifacts for the new version to GH packages, should be a safe action - AFAIK the old version artifacts will still be available in our packages, and only updating `servers/pom.xml` will cause anything to start using the new version

**Comment by amyli123** [comment]
Ah I see! I didn't realize this was a different repo, even though you mentioned this in the feature meeting. My bad. 
This looks good to me; @EverlawKilian would you like to take a look too?

**Comment by EverlawKilian** [comment]
LGTM

**Pull request merged**